java-product-catalog-rest-api/readme.md

```diff
@@ -19,8 +19,8 @@ Please refer to the Choreo documentation under the [Develop a REST API](https://
 #### Use the following build config when creating this component in Choreo:
 
 - Build Preset: **Dockerfile**
```

Comment:
Could you please change this to build pack?

Reply:
Will do chameera